### ISSUE TITLE
[CB] requesting only last token per default

### ISSUE THREAD

**Issue by yannicks1** [vllm-project/vllm-spyre#254]
### [CB] requesting only last token per default

To minimize data transfer, we only want to request the last tokens logits instead of the the entire prompts for CB.
The flag `only_last_token` in the fms forward pass should enable this, however, for right padding we get different results. This branch is to debug this issue in fms and is eventually to be merged when support in fms is established.

-> needs changes in fms and Spyre compiler, `vllm-spyre` code is ready.

related PR: https://github.com/vllm-project/vllm-spyre/pull/253

**Comment by yannicks1** [comment]
#464 has been merged as a intermediate fix/stepping stone. Performance wise not critical anymore to return precisely one token.
